app/authenticated/account/template.hbs

```diff
@@ -0,0 +1,13 @@
+{{#page-layout/header as |header|}}
```

Comment:
Really like this component, nice place for other 'page layout' type components too 👍